tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_common.yml

```diff
@@ -1,6 +1,6 @@
 - name: Disable firewalld before LDAP and Subsystem installation
   systemd: state=stopped name=firewalld
-  when: ansible_distribution == "RedHat"
+  when: ansible_distribution == "RedHat" and ansible_distribution_version <= '7.5'
```

Comment:
this code is never going to run because it will be fedora.Let's keep it as it is.

Reply:
@geetikakay okay. Thanks I'll make the changes. :smile: